Source/Orts.Simulation/Common/Scripting/ScriptManager.cs

```diff
@@ -34,22 +35,18 @@ public class ScriptManager
     {
         readonly Simulator Simulator;
         readonly IDictionary<string, Assembly> Scripts = new Dictionary<string, Assembly>();
-        static readonly ProviderOptions ProviderOptions = new ProviderOptions(Path.Combine(new Uri(Path.GetDirectoryName(Assembly.GetExecutingAssembly().CodeBase)).LocalPath, "roslyn", "csc.exe"), 10);
-        static readonly CSharpCodeProvider Compiler = new CSharpCodeProvider(ProviderOptions);
-
-        static CompilerParameters GetCompilerParameters()
+        static readonly string[] ReferenceAssemblies = new[]
         {
-            var cp = new CompilerParameters()
-            {
-                GenerateInMemory = true,
-                IncludeDebugInformation = Debugger.IsAttached,
-            };
-            cp.ReferencedAssemblies.Add("System.dll");
-            cp.ReferencedAssemblies.Add("System.Core.dll");
-            cp.ReferencedAssemblies.Add("ORTS.Common.dll");
-            cp.ReferencedAssemblies.Add("Orts.Simulation.dll");
-            return cp;
-        }
+            typeof(System.Object).GetTypeInfo().Assembly.Location,
+            typeof(System.Diagnostics.Debug).GetTypeInfo().Assembly.Location,
+            typeof(ORTS.Common.ElapsedTime).GetTypeInfo().Assembly.Location,
+            typeof(ORTS.Scripting.Api.Timer).GetTypeInfo().Assembly.Location,
+            typeof(System.Linq.Enumerable).GetTypeInfo().Assembly.Location,
```

Comment:
GetTypeInfo() doesn't seem to be needed: the Type object returned by typeof() already has an Assembly property.

Reply:
Agreed, keep it simple.